hashicorp/boundary-ui
# refactor: 💡 Update Manage Dropdown in auth-methods

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14704

# Description
Updated manage dropdown in auth-methods to reflect HDS Design

## Screenshots (if appropriate)
**Before**
<img width="1643" alt="authemethod before1" src="https://github.com/user-attachments/assets/ee65edf5-9d14-46c5-bb26-0c4d9c25e4d3">
<img width="1643" alt="authmethod before2" src="https://github.com/user-attachments/assets/992855a0-7eb0-4fb2-872f-5c6ef83d0491">
<img width="1798" alt="authmethod after1" src="https://github.com/user-attachments/assets/119d0f49-440a-4218-964f-17b101e38410">
<img width="1798" alt="authmethod after2" src="https://github.com/user-attachments/assets/74fef3c1-ee4e-4193-9cb4-ff6166a071dc">


## How to Test
1. Start up boundary Instance
2. Navigate to auth-methods
3. "Manage" should reflect HDS Design

## Checklist

~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
